app/controllers/candidate_interface/references/review_controller.rb

```diff
@@ -22,6 +22,11 @@ def complete
           render 'show_for_reference_selection' and return
         end
 
+        if !current_application.minimum_references_available_for_selection?
```

Comment:
I'm wondering if we just shouldn't show the radios in this situation and have a continue button. Wdyt?

Reply:
@JR-G is implementing the design improvements suggested by @paulrobertlloyd, which result in this moving elsewhere. Let's revisit in https://github.com/DFE-Digital/apply-for-teacher-training/pull/4824 when that's ready for review!